doc/source/train/examples/pytorch/deepspeed_finetune/ci/tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+python ci/nb2py.py README.ipynb README.py  # convert notebook to py script
+python README.py  # run the converted python script
```

Comment:
```suggestion
python README.py  # run the converted python script
python train.py
```
Maybe include test for also making sure that `train.py` is able to run to completion